rector.php

```diff
@@ -32,5 +31,8 @@
     $rectorConfig->fileExtensions(['php', 'module', 'theme', 'install', 'profile', 'inc', 'engine']);
     $rectorConfig->importNames(true, false);
     $rectorConfig->importShortClasses(false);
-    $parameters->set('drupal_rector_notices_as_comments', true);
+
+    $rectorConfig->singleton(AddCommentService::class, function() {
+        return new AddCommentService(true);
```

Comment:
Should we remove the argument if it's always `true`

Reply:
As i said before, i'd like to disable it is possible. But that could also perhaps be an empty service so we dont need the config normally?